### PR TITLE
update build and run settings for ndpi project

### PR DESCRIPTION
**Pull request by koltiradw**
Hello there! I have removed timeouts in fuzz configurations and added dicts.

### CODE REVIEW

File: projects/ndpi/ndpi-reader-afl++.toml [hunk 1]
```diff
@@ -20,9 +20,9 @@ target = "/nDPI/sydr/load_sydr_ndpi_reader @@"
 jobs = 2
 
 [aflplusplus]
-args = "-t 1000 -i /corpus"
```
Comment (SweetVishnya) [inline projects/ndpi/ndpi-reader-afl++.toml:23]: default timeout is 10ms for afl++, so, there should be some

File: projects/ndpi/Dockerfile [hunk 2]
```diff
@@ -29,8 +29,6 @@ RUN mkdir corpus && mv nDPI/tests/pcap/* corpus
 
 WORKDIR nDPI
 
-RUN git checkout 01622c4c63f2968da101bedff0db0e8578b04a9b
```
Comment (SweetVishnya) [inline projects/ndpi/Dockerfile:32]: checkout latest master commit hash

File: projects/ndpi/process-packet-afl++.toml [hunk 3]
```diff
@@ -20,9 +20,9 @@ target = "/nDPI/sydr/load_sydr_process_packet @@"
 jobs = 2
 
 [aflplusplus]
-args = "-t 1000 -i /corpus"
+args = "-i /corpus -x /ndpi.dict"
```
Comment (SweetVishnya) [inline projects/ndpi/process-packet-afl++.toml:23]: Set some timeout for AFL++

**Review by SweetVishnya — CHANGES_REQUESTED** [review]

File: projects/ndpi/ndpi-reader-afl++.toml (hunk 1 above)
Comment (koltiradw) [reply projects/ndpi/ndpi-reader-afl++.toml:23]: resolved

File: projects/ndpi/Dockerfile (hunk 2 above)
Comment (koltiradw) [reply projects/ndpi/Dockerfile:32]: resolved

File: projects/ndpi/process-packet-afl++.toml (hunk 3 above)
Comment (koltiradw) [reply projects/ndpi/process-packet-afl++.toml]: resolved

**Review by SweetVishnya — APPROVED** [review]
thx!

**Pull request merged**